src/Sarif/RuntimeConditions.cs

```diff
@@ -49,14 +49,14 @@ public enum RuntimeConditions : uint
         ExceptionInstantiatingSkimmers = 0x8000,
 
         // Non-fatal conditions
-        UnassignedNonfatal        = 0x03F00000,
-        RuleWasExplicitlyDisabled = 0x04000000,
-        RuleCannotRunOnPlatform   = 0x08000000,
-        RuleNotApplicableToTarget = 0x10000000,
-        TargetNotValidToAnalyze   = 0x20000000,
-        OneOrMoreWarningsFired    = 0x40000000,
-        OneOrMoreErrorsFired      = 0x80000000,
+        UnassignedNonfatal        = 0x03F0000,
+        RuleWasExplicitlyDisabled = 0x0400000,
+        RuleCannotRunOnPlatform   = 0x0800000,
+        RuleNotApplicableToTarget = 0x1000000,
+        TargetNotValidToAnalyze   = 0x2000000,
+        OneOrMoreWarningsFired    = 0x4000000,
+        OneOrMoreErrorsFired      = 0x8000000,
```

Comment:
>0x8000000 [](start = 36, length = 9)

the idea previously here was to reserve the most significant bits for the non-fatal conditions. Can you please reset the enum values starting with 0x07000000? And set the nonfatal mask to 7FF?

I realize you are just preserving the readability of the previous values. But since we're having to change the literal underlying enum values in any case, would prefer to preserve the convention of reserving 'leftmost' bits for this class of data.

Reply:
I think you mean 0x40000000, which is the next bit down. We could use the high bit with unchecked casts, but that's confusing. I've now shifted the values and masks over one bit instead of four bits.